manage.py

```diff
@@ -1,9 +1,47 @@
 #!/usr/bin/env python
+
```

Comment:
Here.

Reply:
Thank you. I've squashed commits.

Reply:
Okay, now remove the formatting changes.

Reply:
I'm sorry, yes, I assume when I resolved conflicts I didn't finish this issue with `#!/usr/bin/env python`.
Also, I didn't notice just the empty line in the config. At first, I guess you meant about `coverage`. Now I hope the PR is atomic. I understand that `pylint` was added in the dependencies, but it's just one minor change not closely related to this PR.

Reply:
Marking this as resolved.